homework01/test/examples/Main.hs

```diff
@@ -1,15 +1,44 @@
-module Main
-  ( main
-  )
-  where
-
--- doctest
-import qualified Test.DocTest as DocTest
+module Main where
 
+import Basis
+import Test.Hspec
 
 main :: IO ()
-main =
-  DocTest.doctest
-    [ "-isrc"
-    , "src/Basis.hs"
-    ]
+main = hspec $ do
+
+  describe "toDigits" $ do
```

Comment:
Not sure if it's important, but what about the case for numbers starting with `0` like `01234`